tests/test_common.py

```diff
@@ -26,3 +59,60 @@ def test_cross_section_eq():
     assert cx1 == cx2
     cx2.inelastic = 10
     assert cx1 != cx2
+
+
+def test_EventData_copy_and_pickle(evt):
+    evt2 = evt.copy()
+    assert evt2 == evt
+
+    s = pickle.dumps(evt)
+    evt3 = pickle.loads(s)
+
+    assert evt3 == evt
+
+
+class DummyEvent(MCEvent):
+    def __init__(self):
+        hepevt = SimpleNamespace(
```

Comment:
I like the concept of SimpleNamespace. Is it something used only for tests or can I use it for other codes? It's an official language element...

Reply:
Yes, you can use this everywhere. I use it here for a poor-mans mocking, although the Python standard lib also contains a proper mocking library with unittest.mock.